src/extensions/try-convert/Microsoft.DotNet.UpgradeAssistant.Extensions.TryConvert/TryConvertRunner.cs

```diff
@@ -83,6 +85,26 @@ public async Task<UpgradeStepInitializeResult> InitializeAsync(IProject project,
             }
         }
 
+        private void AddResultToContext(IUpgradeContext context, string status, string resultMessage)
```

Comment:
at this point I'd love to see `.AddResult()` as an extension method on `IUpgradeContext` which takes the necessary parameters OR a delegate to populate things in a more custom manner. This code seems to have become boilerplate for every Step